src/printer/smt2/smt2_printer.cpp

```diff
@@ -42,10 +43,12 @@
 #include "smt/command.h"
 #include "theory/bags/table_project_op.h"
 #include "theory/arrays/theory_arrays_rewriter.h"
+#include "theory/bags/table_project_op.h"
```

Comment:
is this supposed to be in this PR?

Reply:
This was accidental from a merge, I'll delete.